### DISCUSSION TITLE
No current Docker images

### DISCUSSION THREAD

**Discussion by strausmann** [simple-login/app#1303] [Self-hosting question]
There are no current Docker images on Dockerhub, is there a reason for this? I am currently testing the 4.6.2 beta.

**Comment by zeroav-org** [comment]
The reason may be that the product was acquired by Proton and much development focus in recent months was on making the products integrate. I suspect that the monetized side of the product will get more love now vs us self-hosted people that cannot be monetized. Perhaps a hybrid approach could be a solution? The way bitwarden do it. Even better would be if I am wrong and overly cynical 🤔

**Reply by strausmann** [reply]
I think you will not be completely wrong. Very sad...

**Reply by nguyenkims** [reply]
Hi please rest assured that we continue to maintain the self hosted version. The https://github.com/simple-login/app/tree/4.6.2-beta will be moved to "release" state (i.e. removing the `-beta`) soon as it seems to be stable.

**Reply by strausmann** [reply]
In the meantime, there are again a few new beta Docker images. But unfortunately still no new stable.
The beta images also have the complete SimpleLogin layout including the footer and the links to simplelogin.io for terms, privacy, imprint and co.

When can we expect a new stable release with a neutral design?

Thanks a lot for your work and this great project

**Reply by strausmann** [reply]
In the meantime, you should include a workaround for the problem with Issue #1203 in the manual.

**Comment by t2noob** [comment]
If you want to try the latest and greatest you can always use the app-ci from docker hub. From what I believe is the most current one. That one gets updated any time there is a push to gihub. But just be willing and able to revert back if there is an issue/bug with it.
